plots/Main.cpp

```diff
@@ -0,0 +1,47 @@
+#include <string>
+#include <iostream>
+#include <source_dir.h>
+
+#ifndef PYTHON_PATH
+#define PYTHON_PATH ""
+#endif
+
+#if defined(_WIN32)
+#include <direct.h>
+#define cross_getcwd _getcwd
+#else
+#include <unistd.h>
+#define cross_getcwd getcwd
+#endif
+
+
+void python(std::string& args)
+{
+    std::string filename;
+    {
+        char curdir[1024];
+        (void)cross_getcwd(curdir, 1024);
+        filename = curdir;
+    }
+#if defined(_WIN32)
+    std::string slash = "\\";
+#else
+    std::string slash = "/";
+#endif
+    filename = MAIN_PATH + slash + "plots" + slash + "python" + slash + "plot.py";
+
+    (void)std::system((PYTHON_PATH + slash + "python3 \"" + filename + "\"" + args).c_str());
```

Comment:
Please support the default python path.
When I use the default python3, it will report `/python3: not found` (an extra slash).